### DISCUSSION TITLE
Please Help Me With This Code Header in shopify to keep the logo out of the picture and the divider line out of the way

### DISCUSSION THREAD

**Discussion by dimasfaiz** [community/community#64832] [Programming Help]
### Body

Is there someone who would help me solve the problem? The image below the line
![tipsyrepublic shop_(iPhone 12 Pro)](https://github.com/community/community/assets/24537417/11fb3964-f6cf-4fa3-95f1-9d6d1b40131d)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by agetbase** [comment]
Can you illustrate the design you have in mind with a simple drawing?

**Reply by dimasfaiz** [reply]
@developersung13 i apologize late reply, sure this image attached, please let me know, the code raises the logo so that it doesn't hit the image and removes the dividing line, please help with the code so that I can input the code directly.
![image illustration](https://github.com/community/community/assets/24537417/cb7836cb-5e9e-4f19-be78-916c846e1ded)

**Comment by agetbase** [comment]
@dimasfaiz Thank you for your response. I've reviewed the **HTML** and **CSS** codes of the website. Firstly, to remove the `divider line`, you can simply delete the `border-bottom` CSS attribute found in the 
```css
.header-wrapper--border-bottom { ... }
```
section as shown in the image below.

![Screenshot 2023-08-26 at 2 15 10 AM](https://github.com/community/community/assets/56868605/c18a1567-2018-44ff-8067-81879ac5ff6e)

---

Additionally, there are various ways to adjust the position of the **logo**. One observation I made is that the height of the logo image used on the page is relatively **large**, causing the header section to expand as well. Thus, you might consider **reducing the height of the logo image** or adjusting the `margin-top` CSS attribute of the `header` to around **-5rem to -2rem**.

![Screenshot 2023-08-26 at 2 20 05 AM](https://github.com/community/community/assets/56868605/1a853de5-57f1-46c6-b8fa-b350801d477d)

**Reply by dimasfaiz** [reply]
@developersung13 hello developer sung 13, can i find in shopify section with this code: .header-wrapper--border-bottom { ... }
?

**Reply by dimasfaiz** [reply]
@developersung13 
![image](https://github.com/community/community/assets/24537417/ecf9d7ad-46f3-4131-9474-31336102ab58)
can i find section code in shopify?
please let me know, thank you

**Comment by agetbase** [comment]
@dimasfaiz The CSS section would best be understood by looking at the actual code of the website. Might I get the repository URL where the website's code is hosted?
